apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -530,6 +550,7 @@ contract DisputableVoting is IForwarderWithContext, DisputableAragonApp {
     * @param _voteId Vote identifier
     * @return True if the vote is within its overrule window
     */
+   // Note: maybe better renamed as `canRepresentativesVote()`?
```

Comment:
It is actually the other way around, during the overrule window representatives cannot vote